src/components/Feed.tsx

```diff
@@ -2,16 +2,32 @@ import React, { useState } from "react";
 import NewPost from "./NewPost";
 import PostList from "./PostList";
 import { Button } from "antd";
-import { useAppSelector } from "../redux/hooks";
+import { useAppDispatch, useAppSelector } from "../redux/hooks";
+import * as types from "../utilities/types";
+import { ArrowLeftOutlined } from "@ant-design/icons";
+import { setDisplayedProfileId } from "../redux/slices/userSlice";
 
 enum FeedTypes {
   MY_FEED,
   MY_POSTS,
   DISCOVER,
+  PROFILE_POSTS,
```

Comment:
not being fully used yet, but will come in in this story and wanted the onclick to do nothing here [https://www.pivotaltracker.com/story/show/177349009](url)